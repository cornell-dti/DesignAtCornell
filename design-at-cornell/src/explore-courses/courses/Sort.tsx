import React from 'react';
import {
  Divider, 
  SortBy
} from '../ExploreCoursesStyles';
import arrow from '../../static/images/down-arrow.png';

const Sort = () => (
  <Divider>
    <SortBy>
      <h6>Sort By &nbsp;</h6>
      <p>recommended</p>
      <img src={arrow} alt="sort" />
    </SortBy>
  </Divider>
)

export default Sort;
