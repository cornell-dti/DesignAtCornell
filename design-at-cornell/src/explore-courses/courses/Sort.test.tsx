import React from 'react';
import renderer from 'react-test-renderer';
import Sort from './Sort';

describe('<Sort />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Sort />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
