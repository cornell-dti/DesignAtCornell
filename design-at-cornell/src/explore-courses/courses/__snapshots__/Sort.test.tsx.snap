// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sort /> renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  height: 80px;
  padding-top: 20px;
  padding-right: 80px;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  height: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 p {
  font-weight: bold;
  font-size: 20px;
  line-height: 23px;
  margin: 0;
  color: #828282;
}

.c1 img {
  width: 21px;
  height: 14px;
  margin-left: 10px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <p>
      Sort By
    </p>
    <img
      alt="sort"
      src={
        Object {
          "ReactComponent": Object {
            "$$typeof": Symbol(react.forward_ref),
            "render": [Function],
          },
          "__esModule": true,
          "default": "sort-triangle.svg",
        }
      }
    />
  </div>
</div>
`;
